src/browsers/index.ts

```diff
@@ -27,7 +27,7 @@ export class BrowserManager {
   private launching: Map<string, Promise<unknown>> = new Map();
   private timers: Map<string, number> = new Map();
 
-  constructor(private config: Config) {}
+  constructor(private config: Config) { }
```

Comment:
Prettier might remove this, just an FYI.

Reply:
prettier applied, thanks!!